### PR TITLE
TK248 -

### PR DESCRIPTION
**Pull request by karuna835**
1. Bundle ID 124 is missing from the program/course dropdown list
2. According to selected program/course the created batches to be shown
3. Appending Bundle ID / Course Id next to course/program name (Eg: CB129 - Integrated Program in Business Analytics, 395 - Genpact - Data Science 201)

### CODE REVIEW

File: app/apis/btcapi/students.php [hunk 1]
```diff
@@ -29,11 +29,20 @@
...
-	$courses = db_query("SELECT course_id AS id, name, sis_id FROM course WHERE sis_id IS NOT NULL ORDER BY course_id;");
+	$course = db_query("SELECT course_id AS id, name, sis_id FROM course WHERE sis_id IS NOT NULL ORDER BY course_id;");
+	if(!empty($course))
+	{
+		foreach($course as $course_data)
+		{
+			$course_data['name'] = "c".$course_data['id']." | ".$course_data['name'];
+			$courses[] = $course_data;
+		}
+	}
```
Comment (kailasb) [inline app/apis/btcapi/students.php:40]: to the looping in angular view

File: app/apis/btcapi/students.php [hunk 2]
```diff
@@ -531,8 +541,27 @@
...
+	
+	$program_batch = db_query("SELECT bundle_id , code as batch_code FROM bootcamp_batches;");
+	$batches_code = [];
+	if(!empty($program_batch))
+	{
+		foreach($program_batch as $batch)
+		{
+			$batches_code['program'][$batch['bundle_id']][] = $batch;
+		}
+	}
```
Comment (kailasb) [inline app/apis/btcapi/students.php:553]: reuse the above code which uses to get batches by bundle_id

**Review by kailasb — CHANGES_REQUESTED** [review]
check feedback

File: app/apis/btcapi/students.php (hunk 1 above)
Comment (karuna835) [reply app/apis/btcapi/students.php]: added changes in angular view

File: app/apis/btcapi/students.php (hunk 2 above)
Comment (karuna835) [reply app/apis/btcapi/students.php]: get batches by using existing code

**Review by kailasb — APPROVED** [review]
changes are proper

**Pull request merged**